src/lib/comms/include/sol-http-server.h

```diff
@@ -54,6 +55,29 @@ extern "C" {
 struct sol_http_server;
 
 /**
+ * @struct sol_http_server_config
+ * @brief
+ *
+ * It's created with sol_http_server_new() and should be later
+ * deleted with sol_http_server_del()
+ */
+struct sol_http_server_config {
+#ifndef SOL_NO_API_VERSION
+#define SOL_HTTP_SERVER_CONFIG_API_VERSION (1)  /**< compile time API version to be checked during runtime */
+    /**
+     * must match #SOL_HTTP_SERVER_CONFIG_API_VERSION at runtime.
+     */
+    uint16_t api_version;
+#endif
+
+    uint16_t port;
```

Comment:
I was thinking about, in this case it's possible consider the use of the default port in case any config is given. What do you think ?